Assets/Monsters/Monster Health/MonsterHP.cs

```diff
@@ -19,7 +19,7 @@ public void TakeDamage(int DamageAmount)
         _currentHP -= DamageAmount;
         if(_currentHP <= 0)
         {
-            Destroy(gameObject);
+           Destroy(gameObject);
```

Comment:
You deleted a space... now this line has the wrong number of spaces, was supposed to be 4 now is 3 :D
But I won't reject the review just for that